src/lib/openapi/spec/project-activity-schema.ts

```diff
@@ -0,0 +1,29 @@
+import type { FromSchema } from 'json-schema-to-ts';
+
+export const projectActivitySchema = {
+    $id: '#/components/schemas/projectActivitySchema',
+    type: 'array',
+    description:
+        'An array of project activities, each containing a date and activity count',
+    items: {
+        type: 'object',
+        additionalProperties: false,
+        required: ['date', 'count'],
+        properties: {
+            date: {
+                type: 'string',
+                description: 'Activity date',
```

Comment:
Should this have a format? E.g. 
```suggestion
                type: 'string',
                format: 'date',
                description: 'Activity date',
```